packages/components/src/InputDate/InputDate.rebuilt.tsx

```diff
@@ -60,7 +60,6 @@ export const InputDateRebuilt = forwardRef<
       <div onClick={onClick}>
         <InputText
           aria-describedby={activatorProps.ariaDescribedBy}
-          aria-invalid={activatorProps.ariaInvalid === "true" ? true : false}
```

Comment:
I am of the opinion that InputText should be responsible for `aria-invalid`, and it should not be left up to a consumer

the reason being that we offer `invalid` and `error`. these also offer the associated styles.

so, the way to make an InputText have all the various aspects that go along with being invalid is thru the `invalid` or `error` props. with that, `aria-invalid` should be derived from those, not provided explicitly.